core/src/dpdk/port.rs

```diff
@@ -411,7 +462,21 @@ impl<'a> PortBuilder<'a> {
     #[allow(clippy::cognitive_complexity)]
     pub fn finish(&mut self, promiscuous: bool, multicast: bool, with_kni: bool) -> Result<Port> {
         let len = self.cores.len() as u16;
-        let conf = ffi::rte_eth_conf::default();
+        let mut conf = ffi::rte_eth_conf::default();
+
+        // turns on receive side scaling if port has multiple cores.
+        if len > 1 {
+            conf.rxmode.mq_mode = ffi::rte_eth_rx_mq_mode::ETH_MQ_RX_RSS;
+            conf.rx_adv_conf.rss_conf.rss_hf =
+                (ffi::ETH_RSS_IP | ffi::ETH_RSS_TCP | ffi::ETH_RSS_UDP | ffi::ETH_RSS_SCTP) as u64
```

Comment:
one additional comment, do we want to drive this listing via function, or constant?

Reply:
which function?

Reply:
Moreso a top-level way to grab this set.